android/build.gradle

```diff
@@ -22,15 +22,14 @@ rootProject.allprojects {
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 27
-    buildToolsVersion '27.0.3'
+    compileSdkVersion 28
 
     defaultConfig {
         minSdkVersion 18
-        targetSdkVersion 27
-        versionCode 5
-        versionName "3.1.1"
-        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+        targetSdkVersion 28
+        versionCode 6
+        versionName "3.1.4"
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
```

Comment:
There are no android tests, 
could `testInstrumentationRunner` be deleted?

Reply:
@mogol as it was included before - I left it as it is. Maybe you would like to write some test ) at some point of time )

Reply:
Does the plugin need any references to SupportLibrary/AndroidX?
Looks like if I remove any references to tests libs, then SupportLibrary/AndroidX are not needed.

Reply:
@mogol  ok you can left it as it is. up to you